Makefile

```diff
@@ -28,15 +36,18 @@ dist: update-docs
 # Spacy is require for testing spacy_to_prodigy
 
 $(VIRTUALENV)/.models:
-	$(VIRTUALENV)/bin/python -m spacy download en_core_web_sm
+	$(VENV_BIN)/python -m spacy download en_core_web_sm
 	touch $@
 
 $(VIRTUALENV)/.deep_learning_models:
-#	$(VIRTUALENV)/bin/python -m spacy download en_trf_bertbaseuncased_lg
+#	$(VENV_BIN)/python -m spacy download en_trf_bertbaseuncased_lg
 	touch $@
 
 $(VIRTUALENV)/.non_pypi_packages:
-	$(VIRTUALENV)/bin/pip install git+https://github.com/epfml/sent2vec.git
+	# Install from local git directory - pip install [address] fails on Windows
+	git clone https://github.com/epfml/sent2vec.git
```

Comment:
Nightmare! :S I am not sure how much we use sent2vec at the moment, maybe we should drop it. @nsorros ?